tests/test_read.py

```diff
@@ -353,3 +353,86 @@ def test_mdvrptw_instance():
     assert_(any(depot.y < 0) for depot in data.depots())
     assert_(any(client.x < 0) for client in data.clients())
     assert_(any(client.y < 0) for client in data.clients())
+
+
+def test_vrpspd_instance():
+    """
+    Tests that reading an VRPSPD instance happens correctly, particularly the
+    backhaul data.
+    """
+    data = read("data/RC101.vrp")
+
+    assert_equal(data.num_locations, 5)
+    assert_equal(data.num_depots, 1)
+    assert_equal(data.num_clients, 4)
+
+    assert_equal(data.num_vehicles, 4)
+    assert_equal(data.num_vehicle_types, 1)
+
+    vehicle_type = data.vehicle_type(0)
+    assert_equal(vehicle_type.num_available, 4)
+    assert_equal(vehicle_type.capacity, 200)
+
+    # The first client is a linehaul client (only delivery, no pickup), and
+    # the second client is a backhaul client (only pickup, no delivery). All
+    # other clients have both delivery and pickup.
+    clients = data.clients()
+
+    assert_(clients[0].delivery > 0 and clients[0].pickup == 0)
+    assert_(clients[1].delivery == 0 and clients[1].pickup > 0)
```

Comment:
Can we make this exact? Now that we only have a few clients we can probably write out what these values need to be exactly.